client/samples/lib/sample/vcenter/vm/list_vms.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2014-2017 VMware, Inc.  All Rights Reserved.
+# SPDX-License-Identifier: MIT
+
+require 'sample/framework'
+require 'vapi'
+require 'com/vmware/vcenter/vapi'
+
+# A sample which demonstrates list of all Virtual Machines in a vCenter Server
+# Sample Prerequisites:
+# 1 vCenter and ESX
+class ListVms < SampleBase
+
```

Comment:
Could you remove this empty line? There shouldn't be an empty line after a class definition. It won't cause any issues, it's only stylistic, but it would be nice to have a clean start with new samples and just have to fix the existing ones.